tests/MimeUploadValidatorTest.php

```diff
@@ -42,7 +42,7 @@ public function testInvalidFileExtensionValidatingMimeType()
         $this->assertFalse($result, 'Load failed because file extension does not match excepted MIME type');
         $this->assertEquals('File is not a valid upload', $errors[0]);
 
-        unlink($tmpFilePath);
+        unlink($tmpFilePath ?? '');
```

Comment:
This will emit a warning whether it's null or an empty string:
> [Warning]: unlink(): No such file or directory

Does it _also_ get a deprecation warning emitted when it's null?

Reply:
Confirmed that this does emit a deprecation warning as well as the error if the value passed in is null.